src/roswire/proxy/bag.py

```diff
@@ -221,6 +226,8 @@ def start(self) -> None:
                     f'__name:={self.__bag_name}']
             if self.__exclude_topics:
                 args += ['-x', shlex.quote(self.__exclude_topics)]
+            if self.__restrict_to_topics:
+                args += ['-e', shlex.quote(self.__restrict_to_topics)]
```

Comment:
I don't think `shlex.quote` here is doing what we want it to do. Right now, let's say for topics `(.*)/move_group/(.*)` it is generating the command as `-e (.*)/move_group/(.*)`, while it should be `-e "(.*)/move_group/(.*)"` to work.

I'm not sure how's best to deal with this. I have the dummy solution of:
```
args += ['-e', '"{}"'.format(shlex.quote(self.__restrict_to_topics))]
```